apps/OboeTester/app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 31
     defaultConfig {
         applicationId = "com.mobileer.oboetester"
-        minSdkVersion 23
+        minSdkVersion 30
```

Comment:
Go back to 23.

Reply:
Does changing to 30 work as expected?

Reply:
Yes, I changed minSdkVersion to 30 to repro the build failure. It failed on slCreateEngine.
With this change, it builds OK on SDK 30.
I just forgot to put minSdkVersion back to 23.